source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -1921,6 +1920,27 @@ std::function<std::unique_ptr<OnProcessingResponse>()> FilterConfig::createOnPro
   };
 }
 
+std::unique_ptr<AttributeBuilder> FilterConfig::createAttributeBuilder(
+    const ExternalProcessor& config,
+    Extensions::Filters::Common::Expr::BuilderInstanceSharedConstPtr builder,
+    Server::Configuration::CommonFactoryContext& context) {
+  if (config.has_attribute_builder()) {
+    auto& factory = Envoy::Config::Utility::getAndCheckFactory<AttributeBuilderFactory>(
+        config.attribute_builder());
+    auto attribute_builder_config = Envoy::Config::Utility::translateAnyToFactoryConfig(
+        config.attribute_builder().typed_config(), context.messageValidationVisitor(), factory);
+    if (attribute_builder_config) {
+      return factory.createAttributeBuilder(*attribute_builder_config, FilterName, builder,
+                                            context);
+    } else {
+      throw EnvoyException("Failed to create attribute builder");
```

Comment:
ahh, Envoy should no throw on the data path. 
Log error is fine.  

If we want to capture this config error, one way to do it is to do it in filter factory creation time, if applicable.